utils/__init__.py

```diff
@@ -18,6 +18,9 @@ def get_word_count(search_string):
         result[w] = count + 1
```

Comment:
could use something like this:
```
def search_db(cursor, word):
    cursor.execute(
        """
        SELECT DISTINCT doc_index.title, doc_index.url, page_ranks.rank
        FROM lexicon, doc_index, inverted_index, page_ranks
        WHERE
            lexicon.id = inverted_index.word_id AND
            inverted_index.doc_id = doc_index.id AND
            inverted_index.doc_id = page_ranks.doc_id
            AND lexicon.word = ?
        ORDER BY page_ranks.rank
        """,
        (word,),
    )
    result = cursor.fetchall()
    return result
```

and then call it in `run.py`:
`orderedURLS = search_db(cursor=cursor, word=search_string)`